pt_os_web_portal/backend/helpers/finalise.py

```diff
@@ -164,3 +166,41 @@ def do_firmware_update():
         "touch /tmp/.com.pi-top.pi-topd.pt-poweroff.reboot-on-shutdown",
         timeout=10,
     )
+
+
+def get_non_ap_ip_addresses() -> List:
+    ips = list()
+    for iface in ["wlan0", "eth0", "ptusb0"]:
+        ip = get_internal_ip(iface)
+        if ip.replace("Internet Addresses Not Found", ""):
+            ips.append(ip)
+    logger.info(f"Device IPs: {ips}")
+    return ips
+
+
+def disable_ap_mode() -> None:
+    logger.info("Function disable_ap_mode()")
+    try:
+        run_command("/usr/bin/wifi-ap-sta disable", check=False, timeout=20)
+    except Exception as e:
+        logger.error(f"disable_ap_mode(): {e}")
+
+
+def on_same_network(request) -> Dict:
+    logger.info("Function on_same_network()")
+    on_same_network = False
+    pi_top_ip = ""
+    client_ip = request.remote_addr.split(":")[-1]
```

Comment:
do we need to consider ipv6?

Reply:
yeah, I think we need to support ipv6 now that you mention it